bundle/bundle.go

```diff
@@ -53,6 +53,7 @@ type Bundle struct {
 // SignaturesConfig represents an array of JWTs that encapsulate the signatures for the bundle.
 type SignaturesConfig struct {
 	Signatures []string `json:"signatures,omitempty"`
+	Plugin     string   `json:"plugin",omitempty"`
```

Comment:
Rather than storing just the string, could this not store the actual verifier? That way we wouldn't need to keep that global verifier map around.

Reply:
Hmm no the config wouldn't be the right place for that. Just don't feel too comfortable with that global map, but maybe it's alright.

Reply:
AFAICT it's basically the same thing that's done with custom built-in functions and runtime plugins.  You register them at a global level, and then use/enable them by the key they're registered under.